CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.1.1 (27.10.2020)
```

Comment:
Nah. I think it's better not to bump the package version because of the changes in README.

It's fair to users, because they may decide to update to the latest version and get... nothing. Because there're no changes in the codebase.

Reply:
Good point!